LoggerFrameworkProjectmain/Logger.swift

```diff
@@ -44,6 +45,7 @@ class Logger {
             print(logMessage )
             break
         }
+        return logMessage
     }
     func veryHigh(_ message: String){
```

Comment:
I didnt see tests for all of these. Please share your code coverage when updating or creating a PR.

Reply:
Okay, sure